package.json

```diff
@@ -1,46 +1,31 @@
 {
-  "name": "opendistro-trace-analytics",
-  "version": "0.0.4",
+  "name": "opendistro-trace-analytics-kibana",
+  "version": "1.12.0.0",
   "kibana": {
-    "version": "7.9.1",
+    "version": "7.10.0",
     "templateVersion": "1.0.0"
   },
   "license": "Apache-2.0",
+  "scripts": {
+    "kbn": "node ../../scripts/kbn",
+    "lint": "eslint .",
+    "test:cypress": "cypress run",
+    "build": "yarn plugin-helpers build",
+    "plugin-helpers": "node ../../scripts/plugin_helpers"
+  },
   "dependencies": {
-    "@types/react-plotly.js": "^2.2.4",
-    "cypress": "^5.6.0",
     "lodash": "^4.17.20",
     "plotly.js": "^1.54.1",
-    "react": "^16.12.0",
     "react-graph-vis": "^1.0.5",
     "react-plotly.js": "^2.4.0"
   },
-  "scripts": {
-    "kbn": "node ../../scripts/kbn",
-    "lint": "eslint .",
-    "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
-    "build": "plugin-helpers build"
-  },
   "devDependencies": {
-    "@elastic/eslint-config-kibana": "link:../../packages/eslint-config-kibana",
-    "@elastic/eslint-import-resolver-kibana": "link:../../packages/kbn-eslint-import-resolver-kibana",
-    "@kbn/plugin-helpers": "link:../../packages/kbn-plugin-helpers",
+    "cypress": "^5.0.0",
```

Comment:
Np: I think we can get rid of this as well, and use the cypress in Kibana, need to change the script for running cypress as well